.github/workflows/release-artifacts.yml

```diff
@@ -0,0 +1,161 @@
+name: Release artifacts
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Can we add a `workflow_dispatch` here, if this workflow fails then we would be able to trigger that manually with a workflow dispatch.

Reply:
Added.